src/base_station/main.cpp

```diff
@@ -62,6 +62,9 @@ unsigned int map_texture_id;
 // Save the start time so we can use get_ticks.
 std::chrono::high_resolution_clock::time_point start_time;
 
+// Bools to prevent holding down some buttons
+bool upScroll, downScroll, shiftScroll;
```

Comment:
Do these prevent me from scrolling multiple lines if I hold the up or down keys, for instance? I would expect it to continue to scroll if I held those keys.